Xero.NetStandard.OAuth2/Model/PayrollUk/EarningsRate.cs

```diff
@@ -125,8 +119,43 @@ public enum EarningsTypeEnum
             /// Enum TipsNonDirect for value: Tips(Non-Direct)
             /// </summary>
             [EnumMember(Value = "Tips(Non-Direct)")]
-            TipsNonDirect = 15
+            TipsNonDirect = 15,
 
+            /// <summary>
+            /// Enum StatutoryAdoptionPayNonPensionable for value:
+            /// </summary>
+            [EnumMember(Value = "Statutory Adoption Pay (Non Pensionable)")]
+            StatutoryAdoptionPayNonPensionable = 16,
+
+            /// <summary>
+            /// Enum StatutoryMaternityPayNonPensionable for value:
+            /// </summary>
+            [EnumMember(Value = "Statutory Maternity Pay (Non Pensionable)")]
+            StatutoryMaternityPayNonPensionable = 17,
+
+            /// <summary>
+            /// Enum StatutoryPaternityPayNonPensionable for value:
+            /// </summary>
+            [EnumMember(Value = "Statutory Paternity Pay (Non Pensionable)")]
+            StatutoryPaternityPayNonPensionable = 18,
+
+            /// <summary>
+            /// Enum StatutoryParentalBereavementPayNonPensionable for value:
+            /// </summary>
+            [EnumMember(Value = "Statutory Parental Bereavement Pay (Non Pensionable)")]
+            StatutoryParentalBereavementPayNonPensionable = 19,
+
+            /// <summary>
+            /// Enum StatutorySharedParentalPayNonPensionable for value:
+            /// </summary>
+            [EnumMember(Value = "Statutory Shared Parental Pay (Non Pensionable)")]
+            StatutorySharedParentalPayNonPensionable = 20,
+
+            /// <summary>
+            /// Enum StatutorySickPayNonPensionable for value:
+            /// </summary>
+            [EnumMember(Value = "Statutory Sick Pay (Non Pensionable)")]
+            StatutorySickPayNonPensionable = 21
         }
```

Comment:
Hi @andyatwebscale - Making sure you are aware these are now available in `3.17.0`

https://github.com/XeroAPI/Xero-NetStandard/blob/master/Xero.NetStandard.OAuth2/Model/PayrollUk/EarningsRate.cs

They are part of a larger PR as described in the release notes, but this contribution was much appreciated.. 

DM me on twitter for some Xero swag if your keen!
> https://twitter.com/serknight_